src/kaltura-player.ts

```diff
@@ -854,6 +859,10 @@ export class KalturaPlayer extends FakeEventTarget {
         this._reset = false;
       }
     });
+    this._eventManager.listen(this, 'prevent_seek', (e: FakeEvent) => {
```

Comment:
hmmm
should ideally be a const exported from the plugin, but then we'll have a circular dependency...